scripts/utils/cliContextUtils.ts

```diff
@@ -0,0 +1,62 @@
+import type { RequestContext } from '@lokalise/fastify-extras'
+import { generateMonotonicUuid } from '@lokalise/id-utils'
+import type { CommonLogger } from '@lokalise/node-core'
+import type z from 'zod'
+import { type AppInstance, getApp } from '../../src/app.js'
+
+export const getArgs = () => {
+  return process.argv.reduce((args: Record<string, unknown>, arg) => {
+    if (arg.startsWith('--')) {
+      const [key, value] = arg.slice(2).split('=')
+      if (key) {
+        args[key] = value ?? true
+      }
+    } else if (arg.startsWith('-')) {
+      for (const flag of arg.slice(1)) {
+        args[flag] = true
+      }
+    }
+    return args
+  }, {})
+}
+
+export const createCliContext = async <Arguments>(
```

Comment:
can you also add sample script using this, and also document the usage in the readme?

Reply:
Added both